tools/rebasehelpers/util/git.go

```diff
@@ -9,25 +9,87 @@ import (
 	"strings"
 )
 
-var UpstreamSummaryPattern = regexp.MustCompile(`UPSTREAM: (revert: )?(([\w\.-]+\/[\w-\.-]+)?: )?(\d+:|<carry>:|<drop>:)`)
-
-// supportedHosts maps source hosts to the number of path segments that
-// represent the account/repo for that host. This is necessary because we
-// can't tell just by looking at an import path whether the repo is identified
-// by the first 2 or 3 path segments.
-//
-// If dependencies are introduced from new hosts, they'll need to be added
-// here.
-var SupportedHosts = map[string]int{
-	"bitbucket.org":     3,
-	"cloud.google.com":  2,
-	"code.google.com":   3,
-	"github.com":        3,
-	"golang.org":        3,
-	"google.golang.org": 2,
-	"gopkg.in":          2,
-	"k8s.io":            2,
-	"speter.net":        2,
+var (
+	UpstreamSummaryPattern = regexp.MustCompile(`^UPSTREAM: (revert: )?(([\w\.-]+\/[\w-\.-]+)?: )?(\d+:|<carry>:|<drop>:)`)
+	BumpSummaryPattern     = regexp.MustCompile(`^bump[\(\w].*`)
+
+	// patchRegexps holds regexps for paths inside vendor dir that are allowed to be patched directly.
+	// It must corresponds to published repositories.
+	PatchRegexps = []*regexp.Regexp{
+		regexp.MustCompile("^k8s.io/.*"),
```

Comment:
actually only k8s.io/kubernetes. Staging dirs also have that prefix. `k8s.io/{utils,kube-openapi,gengo,....}` must not be patched.

Reply:
yeah, looks like we only publish staging dirs, this should match that list

Reply:
done in https://github.com/openshift/origin/compare/ee8636db3b0609dfa40c81534c2fd4303769d6e7..80cfb21aad257b5dadcbc9893f2573e05b5f8613

Reply:
Technically not even the staging dirs, but just kube. Your linked commit looks good.